src/lib/components/layout/about.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+  import PrimaryButton from "../library/primary-button.svelte";
+
+  const items = [
+    "CDE Adoption Journeys",
```

Comment:
minor: use small letters for all words that are not at the beginning or proper names (see Talias comment)